db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -1556,6 +1557,14 @@ private int getConnFromDatabaseName() throws DRDAProtocolException
             p.put(Attribute.DRDA_KERSEC_AUTHENTICATED, Boolean.toString(gssContext.isEstablished()));
         }
 
+        if (database.securityMechanism == CodePoint.SECMEC_TOKEN) {
+            p.put(Attribute.DRDA_SECMEC,
```

Comment:
can we log some things about this? like securityMechanism or pluginName?

Reply:
added logging for parsed authenticator and token. securityMechanism already logged, when it's parsed.